docs/manual/called-methods-checker.tex

```diff
@@ -6,8 +6,12 @@
 of the form ``call method A before method B''. For the purpose of this
 checker, a method has ``definitely been called'' if it is \emph{invoked}:
 a method that might never return or that might throw an exception has
-a method that might never return or that might throw an exception has
 definitely been called on every path after the call, including exceptional paths.
+The checker also assumes that the program is free of null-pointer dereferences:
+for example, a method that is called on an object that is \<null> at run time
+is considered to have been invoked.
+(You can verify that all pointer dereferences are safe
+by running the Nullness Checker (Section~\ref{nullness-checker}).)
```

Comment:
```suggestion
The checker also assumes that the program is free of null-pointer dereferences.
You can verify that all pointer dereferences are safe
by running the Nullness Checker (Section~\ref{nullness-checker}).
```

How about just this?  Maybe we want @mernst's feedback here?

Reply:
That's definitely shorter and sweeter. I'm okay with this version.

I agree getting @mernst's feedback on this would be useful.